db/post_migrate/20250227142351_migrate_sidekiq_jobs.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
-# Delete this migration instead of promoting it (along with the
-# `SidekiqMigration` class)
+# TODO: Remove this when releasing Discourse 3.6
```

Comment:
Before or after releasing Discourse 3.6? 😅

Reply:
Ah that’s not clear :sweat_smile: I mean we should drop this once it’s been included with Discourse 3.5, so when planning the 3.6 release? (or we could drop it just after the 3.5 release I guess) Anyway, I’ll rewrite this to make it clearer :grin: